src/CallingExtensions.js

```diff
@@ -92,6 +92,13 @@ class CallingExtensions {
     });
   }
 
+  navigateToRecord(data) {
```

Comment:
Does it mean that it will redirect to the contact automatically? What's the use case?

Reply:
This redirects to the contact that the third party sends to HubSpot. Related to [#237](https://git.hubteam.com/HubSpot/calling-extensions-fe/pull/237)